lib/autoproj/cli/ci.rb

```diff
@@ -159,11 +154,19 @@ def consolidated_report
                     {}
                 end
 
-                build_report = JSON.load(File.read(@ws.build_report_path))
+                report_path =
+                    if File.file?(@ws.build_report_path)
+                        @ws.build_report_path
+                    elsif File.file?(@ws.update_report_path)
+                        @ws.update_report_path
+                    end
+                return unless report_path
+
+                build_report = JSON.load(File.read(report_path))
                 packages = build_report['build_report']['packages']
```

Comment:
This is different depending on whether the report is a build report or import report

Reply:
Maybe add a test case/;

Reply:
Oh ... yeah. And that's what you get when there are no tests.

Added relevant tests ... Thanks.